v2/oauth/multiple-frontends-with-separate-backends.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Multiple frontend domains with separate backends
+hide_title: true
+---
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import CoreInjector from "/src/components/coreInjector"
+
+# Multiple frontend domains with separate backends
+
+You can use the following guide if you have a single [**Authentication Service**](/docs/oauth/introduction#authentication-service) that is used by multiple applications.
+In turn, each app has separate **`frontend `** and **`backend`** instances that are served from different domains.
+
+Using the actual **OAuth 2.0** terminology, we can say that each **`frontend`** can be considered a [**Client**](/docs/oauth/introduction#client) and each **`backend`** is a [**Resource Server**](/docs/oauth/introduction#resource-server).
+
+:::info
+This guide assumes that you already have setup and configured **SuperTokens** in your [**Authentication Service**](/docs/oauth/introduction#authentication-service). 
+
+For more information on how to do that please check our [quickstart guide](/docs/thirdparty/introduction).
+:::
+
+## 1. Create the OAuth2 Clients
+
+For each of your **`frontend`** applications you will have to create a separate [**OAuth2 client**](/docs/oauth/introduction#client).
+This can be done by directly calling the **SuperTokens Core** API.
+
+<CoreInjector defaultValue="https://example.com">
+
+```bash
+curl -X POST ^{coreInjector_uri_without_quotes}/recipe/oauth2/admin/clients \
```

Comment:
in this issue: https://github.com/supertokens/supertokens-core/issues/582, towards the end of the first comment, under the docs sub points, there is a point saying "When creating clients for web apps, we will tell people to create clients in a way that doesn't require token exchange endpoint to have auth - so that they can use frontend libs that do pkce.". Please ask @porcellus what this means and make this change here.

Reply:
oh I missed commenting on this, but it basically means adding the `"tokenEndpointAuthMethod": "none"` to the request creating the client.

Reply:
resolved

Reply:
Coming back to this. In our last call, we decided that we should add this depending on the library that we suggest users to use. If the lib is a frontend thing, and we are following PKCE, then this needs to be there, else not. So please revisit this step.